### PR TITLE
Add device history documentation

### PR DESCRIPTION
**Pull request by daltzctr**

### CODE REVIEW

File: source/docs/licensing/licensing.rst [hunk 1]
```diff
@@ -39,6 +39,11 @@ A list of purchased (but unattached) license seats are shown here. Click on the
 
 Once the activation is complete, the license will be downloaded to the device. In the event that Tuner X disconnects from the internet or from the robot before this completes, the license is still activated and available for download the next time Tuner X is connected to the internet/robot.
 
+Activating a License without a Robot
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Assuming that the user has connected Tuner X to the robot, the device will be visible in the :guilabel:`Device History` page for licensing while disconnected. For information on this process, visit the relevant :ref:`licensing from device history <docs/tuner/device-history:licensing from device history>` section.
```
Comment (bhall-ctre) [inline source/docs/licensing/licensing.rst:45]: "Once the user has connected Tuner X to the robot, the device will be available in the..."

Also, maybe put the link for the device history page on "licensing while disconnected" and remove the last sentence?

File: source/docs/tuner/device-history.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Device History
+==============
+
+Users can access a list of past devices connected to Tuner X and license them via the :guilabel:`Device History` page. This is accessible from the left-hand sidebar. This list is not automatically refreshed, but users can refresh it by refreshing the page with the refresh icon in the top-right of the page.
```
Comment (bhall-ctre) [inline source/docs/tuner/device-history.rst:4]: "but users can refresh it by ~~refreshing the page with~~ **pressing** the refresh icon in the top-right of the page."

File: source/docs/tuner/device-history.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Licensing from Device History
+-----------------------------
+
+Users can activate a license for a disconnected device by clicking on the device in the Grid. Then, select the "PRO" icon at the bottom right of the device card.
+
+.. image:: images/device-history-pro-location.png
+   :width: 600
+   :alt:
+
+Then, the user can activate a license for the device like normal. Once the device license has been activated, the user still needs to connect Tuner X to the robot to transfer the activated license to the device.
```
Comment (bhall-ctre) [inline source/docs/tuner/device-history.rst:19]: "Then," -> "From there,"

**Review by bhall-ctre — CHANGES_REQUESTED** [review]

File: source/docs/tuner/device-history.rst [hunk 4]
```diff
@@ -0,0 +1,26 @@
+Device History
+==============
+
+Users can access a list of past devices connected to Tuner X and license them via the :guilabel:`Device History` page. This is accessible from the left-hand sidebar. This list is not automatically refreshed, but users can refresh it by pressing the refersh icon in the top-right of the page.
```
Comment (bhall-ctre) [inline source/docs/tuner/device-history.rst:4]: "refersh" typo

**Review by bhall-ctre — CHANGES_REQUESTED** [review]

File: source/docs/tuner/device-history.rst (hunk 4 above)
Comment (daltzctr) [reply source/docs/tuner/device-history.rst:4]: spell check, you exist for a reason

File: source/docs/licensing/licensing.rst [hunk 5]
```diff
@@ -39,6 +39,11 @@ A list of purchased (but unattached) license seats are shown here. Click on the
 
 Once the activation is complete, the license will be downloaded to the device. In the event that Tuner X disconnects from the internet or from the robot before this completes, the license is still activated and available for download the next time Tuner X is connected to the internet/robot.
 
+Activating a License without a Robot
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Assuming that the user has connected Tuner X to the robot, the device will be available in the :guilabel:`Device History` page for :ref:`licensing while disconnected <docs/tuner/device-history:licensing from device history>`.
```
Comment (bhall-ctre) [inline source/docs/licensing/licensing.rst:45]: I don't really care for the wording of "Assuming that the user has connected Tuner X to the robot, the device will be available", trying to figure out a better way to get across the idea that if Tuner X has ever been connected to the robot, all devices that were present on the robot at the time Tuner X was connected are now in the Device History page.

Comment (daltzctr) [reply source/docs/licensing/licensing.rst:45]: "Devices that have been viewed in Tuner X at least once will be visible in Device History"?

Comment (bhall-ctre) [reply source/docs/licensing/licensing.rst:45]: Replace "viewed in" with "seen by" and "visible" with "available" and I think that should be fine.

**Pull request merged**